common-util/src/main/java/org/sunbird/common/request/BaseRequestValidator.java

```diff
@@ -363,6 +364,73 @@ public void validateSearchRequest(Request request) {
               ResponseCode.dataTypeError.getErrorMessage(), JsonKey.FILTERS, "Map"),
           ResponseCode.CLIENT_ERROR.getResponseCode());
     }
+    if (request.getRequest().containsKey(JsonKey.FILTERS)
+        && ((request.getRequest().get(JsonKey.FILTERS) instanceof Map))) {
+      Map<String, Object> map = (Map<String, Object>) request.getRequest().get(JsonKey.FILTERS);
+
+      map.forEach(
```

Comment:
Reduce number of lines by using different method approach. create different-2 method for each validation.